src/index.html

```diff
@@ -1,17 +1,118 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="text">Surname:</label>
+          <input type="text" required id="text">
+        </div>
+        <div class="form-field">
+          <label>
+          Name:
+            <input type="text" required>
+          </label>
+
+        </div>
+        <div class="form-field">
+          <label>How old are You?
+            <input type="number" value="12"
+             min="1" max="100" required>
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="date">Full date of birth:</label>
+          <input type="date" required id="date">
+        </div>
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement</label>
+          <input type="checkbox" required id="terms">
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div class="form-field">E-mail:
+          <label for="email"></label>
+          <input type="email" name="email"
+          placeholder="email@example.com"
+          required id="email">
```

Comment:
EACH one of the attributes, also in these cases the closing bracket should be on a newline too:
```suggestion
          <input
            type="email"
            name="email"
            placeholder="email@example.com"
            required id="email"
         >
```